samples/Samples.Server/Startup.cs

```diff
@@ -34,6 +35,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IChat, Chat>();
 
             MicrosoftDI.GraphQLBuilderExtensions.AddGraphQL(services)
+                .AddSubscriptionDocumentExecuter()
```

Comment:
Examples are "completely" fixed in #693 and will be merged for v6.

Reply:
Oh nice! 💯
I anticipate a slight conflict with this addition but nothing major.

Reply:
Yes. That is why I needed an extra time for review.